### PR TITLE
clarify download bigdl

### PR DESCRIPTION
**Pull request by dding3**

### CODE REVIEW

**Comment by yiheng** [comment]
why we need it?

**Comment by dding3** [comment]
There is a user follow this README to try to run notebook, but he didn't know where to set BigDL_Home and download the file and asked this in google usergroup. So i think it may be better if we clarify that.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Step-by-step Deep Leaning Tutorials on Apache Spark using [BigDL](https://github
 
 ### Start Jupyter Server
 
+* Download BigDL 0.1.1([linux64](https://repo1.maven.org/maven2/com/intel/analytics/bigdl/dist-spark-2.1.0-scala-2.11.8-linux64/0.1.1/dist-spark-2.1.0-scala-2.11.8-linux64-0.1.1-dist.zip), [mac](https://repo1.maven.org/maven2/com/intel/analytics/bigdl/dist-spark-2.1.0-scala-2.11.8-mac/0.1.1/dist-spark-2.1.0-scala-2.11.8-mac-0.1.1-dist.zip)) and unzip file.
```
Comment (yiheng) [inline README.md]: please update the mac download link. The original is stale.

**Comment by dding3** [comment]
Updated mac download link

**Pull request merged**